src/profile-logic/transforms.js

```diff
@@ -600,3 +599,68 @@ export function focusFunction(
     });
   });
 }
+
+export function restoreAllFunctionsInCallNodePath(
```

Comment:
This should have a comment about the fact that there could be multiple matches, but this function only returns only *one* possible match - the first match that it finds. It doesn't try to find the "heaviest" one either (i.e. the one that is displayed first in the calltree because it has the most samples under it).